### ISSUE TITLE
Sizing option

### ISSUE THREAD

**Issue by dahlia** [dalinaum/IRCCloud#2]
Upscaling (and downscaling) texts would help its accessibility.  <kbd>⌘+</kbd>/<kbd>^+</kbd> (and <kbd>⌘-</kbd>/<kbd>^-</kbd>) should be suitable hotkey for it.

**Comment by dalinaum** [comment]
Only renderer thread process can change the zoom factor, and main thread process cannot change it. Unfortanetly, I failed in my attempt to inject some javascript stuffs into main thread that runs IRCCloud. I'm surveying how to implement this feature easily. I will solve this issue later.

**Comment by dahlia** [comment]
Cool!
